decoders/ps2/pd.py

```diff
@@ -62,65 +81,87 @@ def reset(self):
 
     def start(self):
         self.out_ann = self.register(srd.OUTPUT_ANN)
-
-    def putb(self, bit, ann_idx):
-        b = self.bits[bit]
-        self.put(b.ss, b.es, self.out_ann, [ann_idx, [str(b.val)]])
-
-    def putx(self, bit, ann):
+        self.out_py = self.register(srd.OUTPUT_PYTHON)
+
+    def metadata(self,key,value):
+        if key == srd.SRD_CONF_SAMPLERATE:
+            self.samplerate = value
+
+    def get_bits(self, n, edge:'r', timeout=100e-6):
```

Comment:
would suggest adding a note on why a hardcoded timeout value is needed here, and why its value should be 100E-6